python/pyspark/context.py

```diff
@@ -83,40 +130,6 @@ class SparkContext(object):
     def __init__(self, master=None, appName=None, sparkHome=None, pyFiles=None,
                  environment=None, batchSize=0, serializer=PickleSerializer(), conf=None,
                  gateway=None, jsc=None, profiler_cls=BasicProfiler):
-        """
```

Comment:
I removed `__init__` because the doc is already pretty intuitive about that:
![Screen Shot 2020-10-26 at 8 34 03 PM](https://user-images.githubusercontent.com/6477701/97167743-aee4e080-17ca-11eb-9730-e0fc89b41e42.png)

Reply:
So the constructor param descriptions get moved to the class doc?